web/client/utils/ogc/WFS/__tests__/base-test.js

```diff
@@ -0,0 +1,43 @@
+import expect from 'expect';
+import {isGeometryType} from '../base';
```

Comment:
can you maybe tests also other functions even if they are not part of the PR?

otherwise no one will do it